spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe 'Ingredients Index' do
+  it 'visitor views a list of ingrediets with name and cost' do
+    ingrediet1 = Ingredient.create(name: 'Ground Beef', cost: 2)
+    ingrediet2 = Ingredient.create(name: 'Salt', cost: 4)
+
+    visit "/ingredients"
+
+    # save_and_open_page
+
+    expect(page).to have_content(ingrediet1.name)
```

Comment:
These assertions are good, but could be more robust if you anchored them to text they should appear next to. You've set up your view well with labels next to each of these attributes, so you could change your assertions to `    expect(page).to have_content("Cost: #{ingrediet1.cost}")`

This will make sure that this data is located in the correct place, and is meaningfully rendered on the page. Also, if the cost were `7` and there happened to be a footer on the page that happened to include a 7, you could get a false positive by just asserting for the presence of `7`. So it's better to be specific about where the 7 appears and assert the presence of `Cost: 7`